.travis.yml

```diff
@@ -100,5 +100,16 @@ gemfile:
   - gemfiles/rails_4.2.5.1.gemfile
   - gemfiles/rails_4.2.5.2.gemfile
   - gemfiles/rails_4.2.6.gemfile
+  - gemfiles/rails_4.2.7.gemfile
+  - gemfiles/rails_4.2.7.1.gemfile
+
+matrix:
+  include:
+    - rvm: 2.2.2
+      gemfile: gemfiles/rails_5.0.0.gemfile
```

Comment:
Rails 5.0.0 is not tested in Travis (https://travis-ci.org/appfolio/store_base_sti_class/builds/160123312). The last key override the previous one. ```gemfile``` should be a list here.